Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -102,6 +102,8 @@ class DropDownMenu
   private static _statusPopupRepositionTimer: Nullable<number> = null;
   private static _blurTimer: Nullable<number> = null;
   private static _updateFocus: boolean = true;
+  private static _ignoreMouseEvents: boolean = false;
```

Comment:
`_ignoreMouseEvents` might not be the best name, what about `_ignoreMouseHoverEvents` or something of the sort?

Reply:
The same property is used in the handlers for `mouseover`, `mouseout`, and `mouseleave` so adding a `Hover` seems wrong to me.

Reply:
Aren't those "the hover events"? Anyway, e.g. `click` is not disabled, which seemed weird to me.